packages/react/src/components/Tooltip/Tooltip.js

```diff
@@ -231,7 +231,7 @@ class Tooltip extends Component {
    * @private
    */
   _tooltipId =
-    this.props.id || `__carbon-tooltip_${Math.random().toString(36).substr(2)}`;
+    this.props.tooltipId || `__carbon-tooltip_${Math.random().toString(36).substr(2)}`;
```

Comment:
```suggestion
    this.props.tooltipId || this.props.id || `__carbon-tooltip_${Math.random().toString(36).substr(2)}`;
```

should we still account for users passing in an `id` prop or is it intuitive to reach for `tooltipId`, `tooltipBodyId`, and `triggerId`?